### PR TITLE
OP-1141 Mention presence of new distribution files

### PR DESCRIPTION
**Pull request by mwithi**
See OP-1141 and https://github.com/informatici/openhospital-gui/pull/1960.

### CODE REVIEW

File: doc_admin/AdminManual.adoc [hunk 1]
```diff
@@ -1806,6 +1806,7 @@ NOTE: The _dicom.properties_ file can be automatically generated/overwritten at
 === 3.4 examination.properties
 
 The examination module configuration is set in the _examination.properties_ file:
+Default examination module configuration is available in the _examination.properties.dist_ file:
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]:
```suggestion
A default examination module configuration is available in the _examination.properties.dist_ file:
```

File: doc_admin/AdminManual.adoc [hunk 2]
```diff
@@ -1976,6 +1977,7 @@ NOTE: An application restart is required to apply the modified setting.
 === 3.6 sms.properties
 
 SMS communication module configuration is set in the _sms.properties_ file:
+Default SMS configuration is available in the _sms.properties.dist_ file:
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]:
```suggestion
A default SMS configuration is available in the _sms.properties.dist_ file:
```

File: doc_admin/AdminManual.adoc [hunk 3]
```diff
@@ -2050,7 +2052,8 @@ This file defines the parameters for sending SMS using the Skebby HTTP API.
 [#telemetry-properties]
 === 3.7 telemetry.properties
 
-The Telemetry configuration is set in the telemetry.properties_ file:
+The Telemetry configuration is set in the _telemetry.properties_ file:
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]:
```suggestion
The telemetry configuration is set in the _telemetry.properties_ file.
```

File: doc_admin/AdminManual.adoc [hunk 4]
```diff
@@ -1976,6 +1977,7 @@ NOTE: An application restart is required to apply the modified setting.
 === 3.6 sms.properties
 
 SMS communication module configuration is set in the _sms.properties_ file:
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]:
```suggestion
The SMS communication module configuration is set in the _sms.properties_ file.
```

File: doc_admin/AdminManual.adoc [hunk 5]
```diff
@@ -1806,6 +1806,7 @@ NOTE: The _dicom.properties_ file can be automatically generated/overwritten at
 === 3.4 examination.properties
 
 The examination module configuration is set in the _examination.properties_ file:
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]:
```suggestion
The examination module configuration is set in the _examination.properties_ file.
```

File: doc_admin/AdminManual.adoc [hunk 6]
```diff
@@ -2050,7 +2052,8 @@ This file defines the parameters for sending SMS using the Skebby HTTP API.
 [#telemetry-properties]
 === 3.7 telemetry.properties
 
-The Telemetry configuration is set in the telemetry.properties_ file:
+The Telemetry configuration is set in the _telemetry.properties_ file:
+Default Telemetry configuration is available in the _telemetry.properties.dist_ file:
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]:
```suggestion
A default Telemetry configuration is available in the _telemetry.properties.dist_ file:
```

File: doc_admin/AdminManual.adoc [hunk 7]
```diff
@@ -2082,6 +2085,7 @@ NOTE: An application restart is required to apply the modified setting.
 === 3.8 txtPrinter.properties
 
 Text printer configuration is set in the _txtPrinter.properties_ file:
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]:
```suggestion
The text printer configuration is set in the _txtPrinter.properties_ file.
```

File: doc_admin/AdminManual.adoc [hunk 8]
```diff
@@ -2082,6 +2085,7 @@ NOTE: An application restart is required to apply the modified setting.
...
+Default Text printer configuration is available in the _txtPrinter.properties.dist_ file:
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]:
```suggestion
A default text printer configuration is available in the _txtPrinter.properties.dist_ file:
```

File: doc_admin/AdminManual.adoc [hunk 9]
```diff
@@ -2129,6 +2133,7 @@ NOTE: An application restart is required to apply the modified setting.
 === 3.9 xmpp.properties
 
 XMPP module configuration is set in the _xmpp.properties_ file:
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]:
```suggestion
The XMPP module configuration is set in the _xmpp.properties_ file.
```

File: doc_admin/AdminManual.adoc [hunk 10]
```diff
@@ -2129,6 +2133,7 @@ NOTE: An application restart is required to apply the modified setting.
...
+Default XMPP module configuration is available in the _xmpp.properties.dist_ file:
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]:
```suggestion
A default XMPP module configuration is available in the _xmpp.properties.dist_ file:
```

**Review by mizzioisf — COMMENTED** [review]
I would write:

The XXXX module configuration is set in the _examination.properties_ file.
A default configuration file for this module is available in the _examination.properties.dist_ file:

File: doc_admin/AdminManual.adoc [hunk 11]
```diff
@@ -1805,7 +1805,8 @@ NOTE: The _dicom.properties_ file can be automatically generated/overwritten at
 
 === 3.4 examination.properties
 
-The examination module configuration is set in the _examination.properties_ file:
+The examination module configuration is set in the _examination.properties_ file.
+A default examination module configuration is available in the _examination.properties.dist_ file:
```
Comment (mwithi) [inline doc_admin/AdminManual.adoc]:
```suggestion
A default configuration file for this module is available in the _examination.properties.dist_ file:
```

File: doc_admin/AdminManual.adoc [hunk 12]
```diff
@@ -1975,7 +1976,8 @@ NOTE: An application restart is required to apply the modified setting.
 [#sms-properties]
 === 3.6 sms.properties
 
-SMS communication module configuration is set in the _sms.properties_ file:
+The SMS communication module configuration is set in the _sms.properties_ file.
+A default SMS communication module configuration is available in the _sms.properties.dist_ file:
```
Comment (mwithi) [inline doc_admin/AdminManual.adoc]:
```suggestion
A default configuration file for this module is available in the _sms.properties.dist_ file:
```

File: doc_admin/AdminManual.adoc [hunk 13]
```diff
@@ -2050,7 +2052,8 @@ This file defines the parameters for sending SMS using the Skebby HTTP API.
 [#telemetry-properties]
 === 3.7 telemetry.properties
 
-The Telemetry configuration is set in the telemetry.properties_ file:
+The telemetry configuration is set in the _telemetry.properties_ file.
+A default telemetry configuration is available in the _telemetry.properties.dist_ file:
```
Comment (mwithi) [inline doc_admin/AdminManual.adoc]:
```suggestion
A default configuration file for this module is available in the _telemetry.properties.dist_ file:
```

File: doc_admin/AdminManual.adoc [hunk 14]
```diff
@@ -2081,7 +2084,8 @@ NOTE: An application restart is required to apply the modified setting.
 [#txtprinter-properties]
 === 3.8 txtPrinter.properties
 
-Text printer configuration is set in the _txtPrinter.properties_ file:
+The text printer configuration is set in the _txtPrinter.properties_ file.
+A default text printer configuration is available in the _txtPrinter.properties.dist_ file:
```
Comment (mwithi) [inline doc_admin/AdminManual.adoc]:
```suggestion
A default configuration file for this module is available in the _txtPrinter.properties.dist_ file:
```

File: doc_admin/AdminManual.adoc [hunk 15]
```diff
@@ -2128,7 +2132,8 @@ NOTE: An application restart is required to apply the modified setting.
 [#xmpp-properties]
 === 3.9 xmpp.properties
 
-XMPP module configuration is set in the _xmpp.properties_ file:
+The XMPP module configuration is set in the _xmpp.properties_ file.
+A default XMPP module configuration is available in the _xmpp.properties.dist_ file:
```
Comment (mwithi) [inline doc_admin/AdminManual.adoc]:
```suggestion
A default configuration file for this module is available in the _xmpp.properties.dist_ file:
```

**Comment by mwithi** [comment]
> I would write:
> 
> The XXXX module configuration is set in the _examination.properties_ file. A default configuration file for this module is available in the _examination.properties.dist_ file:

alright, like my suggestions (please check)

**Comment by mizzioisf** [comment]
Ok for me.

**Pull request merged**